Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -142,8 +142,14 @@ namespace Multiplayer
         }
 
         // Start the configured server if it's available
+        AZStd::string projectPath(AZ::Utils::GetProjectPath().c_str());
+        AZStd::replace(projectPath.begin(), projectPath.end(), AZ::IO::WindowsPathSeparator, AZ::IO::PosixPathSeparator);
```

Comment:
There is no reason to substitute the path separators with Posix path separtor when invoking the process launcher command with the project path.
This manual path manipulation should be removed.